CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 # Changelog
 
-## UNRELEASED
+*March 8, 2022*
+
+## v0.6.8
 
 ### Bug Fixes
 
 - (tx, rpc) [cronos#377](https://github.com/crypto-org-chain/cronos/pull) back port bug fixes in ethermint: a) report correct gasWanted in checkTx, b) add returnValue message on tracing.
+- (rpc) [#381](https://github.com/tharsis/ethermint/issues/381) support return value on trace tx response.
```

Comment:
can you fix the changelog above?

I think you removed b) so it should be only "report correct gasWanted in checkTx"